sycl/include/CL/sycl/detail/pi.h

```diff
@@ -60,6 +60,10 @@ using pi_uint64 = uint64_t;
 using pi_bool = pi_uint32;
 using pi_bitfield = pi_uint64;
 
+// For selection of SYCL RT back-end, now manually through the "SYCL_BE"
+// environment variable.
```

Comment:
Does the comment still apply? The environmental variable is not used anymore

Reply:
no, i will remove it. i just took the part from pi.hpp to get feedback on this